deploy/default/kustomization.yaml

```diff
@@ -5,13 +5,8 @@ resources:
 - ../namespace
 - ../rbac
 - ../crds
-- ../operator/ironic
+- ../operator
```

Comment:
we should add ironic here to not modify the current behaviour

Reply:
Ah true, added. Thanks!